CONTRIBUTING.md

```diff
@@ -17,41 +16,59 @@ This guide will help you get started and outline some things you should know whe
   (install [directly](https://www.ruby-lang.org/en/documentation/installation),
   or via a ruby manager like [rvm](https://rvm.io/)
   or [rbenv](https://github.com/sstephenson/rbenv))
-  * Language the tests are written in
+  * language the tests are written in
 * [ShellCheck](https://github.com/koalaman/shellcheck)
-  * Used in the linting process to find common errors in the Bash code
+  * used in the linting process to find common errors in the Bash code
 
 
 ## Setup
 
+* install the [requirements](#requirements)
 * fork and clone the repository to your machine
-* `bundle` to install ruby gems
+* run `bundle` to install ruby gems
+* run `bundle exec rake` to make sure all tests pass
 
 
 ## Testing
 
-* Tests are written in [Cucumber](http://cukes.info/) and [RSpec](http://rspec.info/).
-* All features need to have comprehensive test coverage
-* Source code and test files must pass the linters
+* tests are written in [Cucumber](http://cukes.info/) and [RSpec](http://rspec.info/).
+* all features need to have comprehensive test coverage
+* source code and test files must pass the linters
 
 ```bash
-# rake tasks
-rake         # Run linters and feature tests
-rake format  # Run formatters (fixes some lint errors)
-rake lint    # Run linters
-rake test    # Run feature tests
+# run all tests
+bundle exec rake
+
+# run individual tests
+bundle exec rake lint    # Run linters
+bundle exec rake test    # Run feature tests
 
 # run single scenario/feature
-cucumber -n 'scenario/feature name'
-cucumber [filename][:lineno]
+bundle exec cucumber <filename>[:<lineno>]
+bundle exec cucumber -n '<scenario/feature name>'
 
 # run single scenario/feature while showing the application output
-DEBUG_COMMANDS=true cucumber [filename][:lineno]
+DEBUG_COMMANDS=true bundle exec cucumber <filename>[:<lineno>]
 
 # run features in parallel
-bin/cuke [<folder>...]
+bin/cuke [<folder> | <filename>[:<lineno>]]
+
+# fix formatting issues
+bundle exec rake format  # Run formatters (fixes some lint errors)
 ```
 
+Git Town's [CI server](https://circleci.com/gh/Originate/git-town)
+automatically tests all commits and pull requests,
+and notifies you via email and through status badges in pull requests
+about problems.
+
+
+## Merging
+
+* use Git Town itself to ship your pull request (you'll get write access)
```

Comment:
Yeah I believe this should be removed. Its fine giving write access to other Originators but we shouldn't be handing it out to everyone.